E-Commerce/readme.md

```diff
@@ -46,16 +46,87 @@ erDiagram
 Write SQL queries to complete the following tasks:
 
 - [ ] List all the products whose name contains the word "socks"
+
+```sql
+SELECT * FROM products WHERE product_name ILIKE '%socks%';
+```
+
 - [ ] List all the products which cost more than 100 showing product id, name, unit price, and supplier id
+
+```sql
+SELECT p.id, p.product_name, pa.unit_price, PA.SUPP_ID
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    WHERE pa.unit_price > 100;
+```
+
 - [ ] List the 5 most expensive products
+
+```sql
+SELECT p.id, p.product_name, pa.unit_price, PA.SUPP_ID
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    ORDER BY pa.unit_price DESC
+    LIMIT 5;
+```
+
 - [ ] List all the products sold by suppliers based in the United Kingdom. The result should only contain the columns product_name and supplier_name
-- [ ] List all orders, including order items, from customer named Hope Crosby
+
+```sql
+SELECT p.product_name, s.supplier_name
+    FROM products p INNER JOIN
+    product_availability pa on (p.id = pa.prod_id)
+    INNER JOIN suppliers s on (pa.supp_id = s.id)
+    WHERE s.country = 'United Kingdom';
```

Comment:
CORRECT - stylistically, try and keep the line feeds consistent (i.e. starting the 3rd line with INNER JOIN..., rather than having this at the end of the preceding line), although it is irrelevant for correct execution